app/assets/javascripts/host_checkbox.js

```diff
@@ -120,13 +120,12 @@ function submit_modal_form() {
 }
 
 function build_modal(element, url) {
-  var url = url + "?" + $.param({host_ids: $.foremanSelectedHosts});
   var title = $(element).attr('data-dialog-title');
   $('#confirmation-modal .modal-header h4').text(title);
   $('#confirmation-modal .modal-body').empty()
     .append("<div class='modal-spinner spinner spinner-lg'></div>");
   $('#confirmation-modal').modal();
-  $("#confirmation-modal .modal-body").load(url + " #content",
+  $("#confirmation-modal .modal-body").load(url + " #content",{host_ids: $.foremanSelectedHosts},
```

Comment:
@ohadlevy: I've added some tests.
@dLobatog: Why you removed the `Not yet reviewed` label?